tests/ui/conftest.py

```diff
@@ -17,6 +17,18 @@ def capabilities(capabilities):
     return capabilities
 
 
+@pytest.fixture
+def firefox_options(firefox_options):
+    firefox_options.set_preference(
+        'extensions.install.requireBuiltInCerts', False)
+    firefox_options.set_preference('xpinstall.signatures.required', False)
+    firefox_options.set_preference('extensions.webapi.testing', True)
+    firefox_options.set_preference('testpilot.env', 'local')
+    firefox_options.add_argument('-foreground')
+    firefox_options.log.level = 'trace'
+    return firefox_options
```

Comment:
This is golden, nice "documentation" on how to test this on your own machine as well.

Reply:
Heh, I can add some more details. I think we should actually include a README just for the UI tests.

Reply:
I would very much like that. Feel free to do that as a follow-up though and open an issue about it?